crates/fluvio/src/consumer.rs

```diff
@@ -429,28 +431,31 @@ impl PartitionConsumer {
             .lookup_version(DefaultStreamFetchRequest::API_KEY)
             .unwrap_or((WASM_MODULE_API - 1) as i16);
 
-        if let Some(mut module) = config.wasm_module {
-            if stream_fetch_version < WASM_MODULE_API as i16 {
-                return Err(FluvioError::Other("SPU does not support WASM".to_owned()));
-            }
-
-            if stream_fetch_version < WASM_MODULE_V2_API as i16 {
-                // SmartStream V1
-                debug!("Using WASM V1 API");
-                let wasm = module.wasm.get_raw()?;
-                stream_request.wasm_module = wasm.into_owned();
-            } else {
-                // SmartStream V2
-                debug!("Using WASM V2 API");
-                if stream_fetch_version < GZIP_WASM_API as i16 {
-                    module.wasm.to_raw()?;
+        if let Some(smart_module) = config.smart_module {
+            println!("vers {:?}", stream_fetch_version);
```

Comment:
change to debug like: `debug!(stream_version_version,"version")`

Reply:
good catch